client.go

```diff
@@ -418,6 +418,27 @@ func (client *gocloak) GetRequestingPartyToken(ctx context.Context, token, realm
 	return &res, nil
 }
 
+// GetRequestingPartyPermissions returns a requesting party permissions granted by the server
+func (client *gocloak) GetRequestingPartyPermissions(ctx context.Context, token, realm string, options RequestingPartyTokenOptions) (*[]Permission, error) {
```

Comment:
Could you please make a helper `getRequestingParty(ctx context.Context, token, realm string, options RequestingPartyTokenOptions, output interface{}) error` that can be used in both `GetRequestingPartyToken` and `GetRequestingPartyPermissions` functions